### DISCUSSION TITLE
exclude mailadress from catch-all

### DISCUSSION THREAD

**Discussion by Sp00kje** [modoboa/modoboa#2119] [Q&A] [answered]
Hi all,

i got an couple of domain's with catch-all aliases configured.
now i want to exclude certain addresses from the catch-all in a way that it return a SMTP 55x error "mailbox does not exist" tot the sender.
(these addresses are leaked in an external data breach and are now only receiving spam/phishing)

Does Modoboa have an function for this and if so, how can i configure this?

**Comment by kryskool** [comment, marked as answer]
Hi @The0mega 

You can configure [header_check](http://www.postfix.org/header_checks.5.html) on postfix directly, i use it to reject some spam bot

`header_checks = pcre:/etc/postfix/header_checks`

```
/To:.*you@yourdomain.com/     REJECT I hate spammer
/To:.*me@yourdomain.com/     REJECT I hate spammer
```

I let these rules permanently

Regards,

**Comment by Sp00kje** [comment]
Thanks, i will do it that way.
it aren't that many addresses so it should be doable.

Thanks!
